node/network/availability-recovery/src/lib.rs

```diff
@@ -393,7 +393,7 @@ impl RequestChunksPhase {
 						metrics.on_chunk_request_invalid();
 						self.error_count += 1;
 
-						tracing::debug!(
+						tracing::warn!(
```

Comment:
Warning on invalid behavior can be extremely noisy. Nodes misbehaving is abnormal but within expectations, so we typically don't 'warn' in such cases.

Reply:
Still unaddressed

Reply:
Yep, alert on `on_chunk_request_invalid` would probably be better.